modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavascriptClosureAngularClientCodegen.java

```diff
@@ -64,15 +70,11 @@ public JavascriptClosureAngularClientCodegen() {
 
         typeMapping.put("binary", "string");
 
-        outputFolder = "generated-code/javascript-closure-angular";
```

Comment:
@dinukadesilva thanks for reviewing. Good catch.

@miguelvaladez would you please take a look at the feedback by @dinukadesilva when you've time?